### ISSUE TITLE
error when trying to deploy on vercel

### ISSUE THREAD

**Issue by dantas15** [dantas15/me#8]
## error log
```
/vercel/.cache/puppeteer/chrome/linux-126.0.6478.126/chrome-linux64/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory
```

## why is this happening?
- vercel doesn't have a chromium in order to use puppeteer

**Comment by dantas15** [comment]
## idea to solve it

- https://gist.github.com/hajek-raven/d05204c948f773bbeff311b681a79df8 
- https://github.com/Sparticuz/chromium

**Comment by dantas15** [comment]
Solved in [`a010166`](https://github.com/dantas15/me/commit/a0101663ad328959552926d188e5e7be86e63b91)
